stories/HowDoYouFeel.stories.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
there are now two story files, please clean up. Also, file names must be kebob-case.